AppiumSwiftClient/common/driver.swift

```diff
@@ -106,11 +106,33 @@ public class AppiumDriver: Driver {
     public func getPageSource() throws -> String {
         return try W3CGetPageSource().sendRequest(with: currentSession.id)
     }
-    
+
     @discardableResult public func back() throws -> String {
         return try W3CGoBack().sendRequest(with: currentSession.id)
     }
 
+    @discardableResult public func setImplicitTimeout(timeoutInMillisencods: Int) throws -> String {
+        return try setTimeout(with: TimeoutTypesEnum.implicit, and: timeoutInMillisencods)
+    }
+
+    @discardableResult public func setPageLoadTimeout(timeoutInMillisencods: Int) throws -> String {
+        return try setTimeout(with: TimeoutTypesEnum.pageLoad, and: timeoutInMillisencods)
+    }
+
+    @discardableResult public func setScriptTimeout(currentContext: String = "",
+                                                    timeoutInMillisencods: Int) throws -> String {
+        if currentContext.contains("WEBVIEW") {
```

Comment:
It would be better to skip like this check in client side since Appium's concept is thin clients.

I mean Appium handles contexts/errors etc, then returns proper response if something happened or not implemented yet etc. This is because we'd like to keep compatibility between clients and appium server. For instance, when appium supports `setTimeout` in native context, we want users to update only server side. We won't require to update client side for existing commands as possible.

Reply:
Yeah, your reasoning makes sense. I was unsure how to implement this endpoint as the documentation referred that this timeout is only for Webviews, and after testing it against Native context and Appium server returned a 200, I considered that forcing the client to only execute if matching this condition. 
Now I'm second guessing myself if I got the specs right, as I never used this timeout type before and really didn't know what to expect out of it. Anyway, I'm pushing a new version addressing your feedback.